src/tasks/install.mk

```
@@ -9,7 +9,7 @@ instal%: node_modules bower_components dotfiles
 # Regular npm install
 node_modules: package.json
 	@if [ -e package-lock.json ]; then rm package-lock.json; fi
-	@if [ -e package.json ]; then $(NPM_INSTALL) && $(DONE); fi
+	@if [ -e package.json ]; then mkdir -p node_modules && touch node_modules/.metadata_never_index && $(NPM_INSTALL) && $(DONE); fi
```

We don't all work on macOS.

Worth splitting out into a line before that checks for the operating system and creates the `.metadata_never_index`.

```suggestion
	@if [ -e package.json ] && [[ "$OSTYPE" =~ ^darwin ]]; then mkdir -p node_modules && touch node_modules/.metadata_never_index fi
```

Adding a comment would be helpful here too.

---

true, but the file is harmless on other OSs. i'd much rather keep it simple